packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountWrapper.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import flag from 'cozy-flags'
+
+import ConfigurationTab from './ConfigurationTab'
+import DataTab from './DataTab'
+import KonnectorAccountTabs from './KonnectorAccountTabs'
+import {
+  intentsApiProptype,
+  innerAccountModalOverridesProptype
+} from '../../helpers/proptypes'
+import FlowProvider from '../FlowProvider'
+
+export const KonnectorAccountWrapper = props => {
+  const { component, ...rest } = props
+  const navigate = useNavigate()
+
+  let Component
+  switch (component) {
+    case 'DataTab':
+      Component = DataTab
+      break
+
+    case 'ConfigurationTab':
+      Component = ConfigurationTab
+      break
+
+    default:
+      Component = DataTab
+      break
+  }
```

Comment:
I don't like this kind of API. Why passing a string to translate it in component later? Why not passing directly the component? 🙏